core/constant_inc.php

```diff
@@ -623,4 +623,12 @@
 # Note: mantis ids are defined in schema as "I UNSIGNED", which Adodb maps to
 # the closest integer (4 bytes) type available. As some DBs dont support unsigned
 # types, 2^31 is a safe limit to be used for all.
-define( 'DB_MAX_INT', 2147483647 );
\ No newline at end of file
+define( 'DB_MAX_INT', 2147483647 );
+
+# Databse functional type identifiers.
+define( 'DB_TYPE_UNDEFINED', 0 );
+define( 'DB_TYPE_MYSQL', 1 );
```

Comment:
if going the full way, we should identify two types of usage of this config variable:
- check the type of database, in an abstract way: mysql, mssql, etc. 
these usually would use de db_is_xxx methods
- check the specific driver: mysqli, mssql_native, etc
these new to access the config variable, to perform specific operations based on the driver.